integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -11,10 +11,12 @@ setup() {
 	export KUBECONFIG="$HOME/.kube/config"
 	pod_name="handlers"
 
-	if kubectl get runtimeclass | grep kata; then
+	if kubectl get runtimeclass 2> /dev/null | grep kata; then
```

Comment:
Nit: I appreciate this is a minimal change, but it might be worth making the logic consistent since sometimes the tests are doing `grep kata` and other times `grep -q kata` (probably better).

But looking at the diff, I wonder if this logic could be centralised into a shell function as the `if` test and the variable being set looks seems to be the same each time.

Just a thought.

Reply:
@jodh-intel addressed both comments, ptal